thirdeye/thirdeye-frontend/app/pods/components/rootcause-data-indicator/component.js

```diff
@@ -39,7 +39,12 @@ export default Component.extend({
 
     fetch(`/rootcause/raw?framework=identity&urns=${selectedUrn}`)
       .then(checkStatus)
-      .then(res => setProperties(this, { maxTime: parseInt(res[0].attributes.maxTime[0], 10) }));
+      .then(res => {
+        let maxTime = res[0].attributes.maxTime;
```

Comment:
Are we sure res will always be a defined array of objects with an attributes property? Because if not this may break.